genomic_data_service/rnaseq/matrix.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+
+from flask import Response
+
+from genomic_data_service.reports.csv import CSVGenerator
+from genomic_data_service.searches.requests import make_search_request
+from snosearch.parsers import QueryString
+
+
+def extract_value(expression, key, default=''):
+    value = expression
+    for field in key:
```

Comment:
can `key` be an array, tuple, dict, etc?

Reply:
Using tuple of strings but I suppose it could be any iterable. I guess an argument for type annotation.